test/Conversion/TorchConversionToMLProgram/multiple_functions.mlir

```diff
@@ -0,0 +1,15 @@
+// RUN: torch-mlir-opt %s -convert-torch-conversion-to-mlprogram -split-input-file | FileCheck %s
+
+module {
+  func.func private @f0() -> i64
```

Comment:
A higher number of functions helps trigger the race when multiple threads process them concurrently.